presto-native-execution/presto_cpp/main/common/Configs.cpp

```diff
@@ -79,7 +205,7 @@ int SystemConfig::httpServerHttpsPort() const {
   return requiredProperty<int>(std::string(kHttpServerHttpsPort));
 }
 
-bool SystemConfig::enableHttps() const {
+bool SystemConfig::httpServerHttpsEnabled() const {
```

Comment:
Maybe we should have all bool value naming to be consistently either enable* or *Enabled?